content/sdk/10.js/00.ethers/10.v5/07.utils.md

```diff
@@ -194,64 +113,50 @@ const L2_BASE_TOKEN_ADDRESS = '0x000000000000000000000000000000000000800a';
 const BOOTLOADER_FORMAL_ADDRESS = "0x0000000000000000000000000000000000008001";
 ```
 
-#### Contract deployer
+#### Contract Deployer
 
 ```ts
 const CONTRACT_DEPLOYER_ADDRESS = "0x0000000000000000000000000000000000008006";
 ```
 
-#### L1 messenger
+#### L1 Messenger
 
 ```ts
 const L1_MESSENGER_ADDRESS = "0x0000000000000000000000000000000000008008";
 ```
 
-#### Nonce holder
+#### Nonce Holder
 
 ```ts
 const NONCE_HOLDER_ADDRESS = "0x0000000000000000000000000000000000008003";
 ```
 
 ### Gas
 
-#### `DEFAULT_GAS_PER_PUBDATA_LIMIT`
-
-- Use a large amount of gas per pubdata for signing on layer 2.
-- The amount ensures any reasonable limit is accepted.
-
-::callout{icon="i-heroicons-exclamation-triangle" color="amber"}
-The operator is NOT required to use the actual value and can use any value up to that signed by the user.
-::
+#### Default Gas Per Pubdata Limit
+Use a large amount of gas per pubdata for signing on layer 2. The amount ensures any reasonable limit is accepted.
 
 ```ts
 const DEFAULT_GAS_PER_PUBDATA_LIMIT = 50000;
 ```
 
-#### `REQUIRED_L1_TO_L2_GAS_PER_PUBDATA_LIMIT`
-
+#### Required L1 to L2 Gas Per Pubdata Limit
 The current required gas per pubdata for L1->L2 transactions.
 
 ```ts
 const REQUIRED_L1_TO_L2_GAS_PER_PUBDATA_LIMIT = 800;
 ```
 
-## Functions
+::callout{icon="i-heroicons-exclamation-triangle" color="amber"}
+The operator is NOT required to use the actual value and can use any value up to that signed by the user.
+::
 
-### `applyL1ToL2Alias`
+### Functions
```

Comment:
Please check the page and the headers you are modifying. H3 is the lowest header tag that the Table of Contents will display. Changing this from an H2 to an H3 and then making all the headers below to H4 removes them all from the ToC.
Compare your [Table of Contents](https://zksync-sdk-docs-staging--pr52-update-the-javascrip-t0x4wius.web.app/sdk/js/ethers/v5/utils) to the production [Table of Contents](https://docs.zksync.io/sdk/js/ethers/v5/utils)